### PR TITLE
Compas update

### PR DESCRIPTION
**Pull request by beverlylytle**
Addresses changes in rhino related imports in compas v0.16.3/4.

### What type of change is this?

- [x] Bug fix in a **backwards-compatible** manner.
- [ ] New feature in a **backwards-compatible** manner.
- [ ] Breaking change: bug fix or new feature that involve incompatible API changes.
- [ ] Other (e.g. doc update, configuration, etc)

### Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code._

- [ ] I added a line to the `CHANGELOG.rst` file in the `Unreleased` section under the most fitting heading (e.g. `Added`, `Changed`, `Removed`).
- [ ] I ran all tests on my computer and it's all green (i.e. `invoke test`).
- [ ] I ran lint on my computer and there are no errors (i.e. `invoke lint`).
- [ ] I added new functions/classes and made them available on a second-level import, e.g. `compas_fab.robots.Configuration`.
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [ ] I have added necessary documentation (if appropriate)

### CODE REVIEW

**Comment by beverlylytle** [comment]
Is there a better way to do this?

File: setup.py [hunk 1]
```diff
@@ -11,7 +11,7 @@
 
 requirements = [
     # Until COMPAS reaches 1.0, we pin major.minor and allow patch version updates
-    'compas>=0.16.2,<0.17',
+    'compas>=0.16.3,<0.17',
```
Comment (beverlylytle) [inline setup.py:14]: I think this is necessary because of the change of location of `xtransform` and `xform_from_transformation`.

Comment (gonzalocasas) [reply setup.py:14]: Shouldn't we bump to `0.16.4`?

Comment (beverlylytle) [reply setup.py:14]: Can do.

File: src/compas_fab/ghpython/path_planning.py [hunk 2]
```diff
@@ -6,10 +6,7 @@
 from timeit import default_timer as timer
 
 import compas
-from compas.datastructures import Mesh
```
Comment (gonzalocasas) [inline src/compas_fab/ghpython/path_planning.py:9]: I'm so looking forward for the time when we can get rid of this file. I believe it's only used for vrep planning examples.

**Comment by gonzalocasas** [comment]
I think it looks good, and I agree with the fix being temporary but necessary.

**Review by gonzalocasas — APPROVED** [review]
🚀 LGTM!

**Pull request merged**